tests/replication/tcp/streams/test_events.py

```diff
@@ -241,24 +242,11 @@ def test_update_function_huge_state_change(self) -> None:
         self.assertIsInstance(row.data, EventsStreamEventRow)
         self.assertEqual(row.data.event_id, pl_event.event_id)
 
-        # the state rows are unsorted
-        state_rows: List[EventsStreamCurrentStateRow] = []
-        for stream_name, _, row in received_rows:
-            self.assertEqual("events", stream_name)
-            self.assertIsInstance(row, EventsStreamRow)
-            self.assertEqual(row.type, "state")
-            self.assertIsInstance(row.data, EventsStreamCurrentStateRow)
-            state_rows.append(row.data)
-
-        state_rows.sort(key=lambda r: r.state_key)
-
-        sr = state_rows.pop(0)
-        self.assertEqual(sr.type, EventTypes.PowerLevels)
-        self.assertEqual(sr.event_id, pl_event.event_id)
-        for sr in state_rows:
-            self.assertEqual(sr.type, "test_state_event")
-            # "None" indicates the state has been deleted
-            self.assertIsNone(sr.event_id)
```

Comment:
I'm unsure if I'm now not testing a codepath by doing this?

Reply:
(I think this is because we have magic constants of 100 for both the number of rows to fetch over replication & also the number of rows to combine. We could probably make the latter 150 just to break this and add separate tests?)

Reply:
Done!